### DISCUSSION TITLE
Shortcut to take control

### DISCUSSION THREAD

**Discussion by JbT102** [rustdesk/rustdesk#6178] [General]
Hello,

Is there a way to add a feature for the host to hit a keyboard shortcut that blocks the mouse/keyboard access to the remote user?
This can come in handy if the connection is acting up or if the communication with the remote user is lost and the local user cannot signal it

Thanks in advance,
JET102

**Comment by My1** [comment]
I mentioned something half an eternity ago, that ideally the user on the machine always has priority against a user connecting in, like on anydesk if I control you and you move your mouse I have to wait until you let go for like at least 2 seconds

here it is:
https://github.com/rustdesk/rustdesk/issues/290#issuecomment-1306328935
